tests/queries/0_stateless/01019_alter_materialized_view_consistent.sh

```diff
@@ -39,7 +39,7 @@ function insert_thread() {
         done
         wait
 
-        is_done=$($CLICKHOUSE_CLIENT -q "SELECT countIf(case = 1) > 0 AND countIf(case = 2) > 0 FROM mv;")
+        is_done=$($CLICKHOUSE_CLIENT -q "SELECT countIf(\`case\` = 1) > 0 AND countIf(\`case\` = 2) > 0 FROM mv;")
```

Comment:
This is a backward incompatibility.
I would rater try to fix it if possible. It is interesting why did it work before?